numpy/linalg/tests/test_linalg.py

```diff
@@ -423,25 +423,6 @@ def test_generalized_empty_herm_cases(self):
             exclude={'none'})
 
 
-def dot_generalized(a, b):
```

Comment:
Funny, I had exactly the same thought -- couldn't you use `matmul` instead -- but then realised it was actually different (in the actual expression, I might have used `einsum` instead, but it is not really important at all).